js/google_ima.js

```diff
@@ -1821,14 +1821,12 @@ require("../html5-common/js/utils/utils.js");
             }
             else
             {
-              //End the fake ad here
               _endCurrentAdPod(this.currentAMCAdPod.isRequest);
             }
           }
           else
           {
-            //End the fake ad here
-            _endCurrentAdPod(this.currentAMCAdPod.isRequest);
+            _endCurrentAdPod(true);
```

Comment:
If the ad is an overlay, it'll go through the flow above. The currentIMAAd object will exist.